Vagrantfile

```diff
@@ -0,0 +1,117 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  config.vm.box = "boxcutter/ubuntu1604"
+
+
+  config.vm.network "forwarded_port", guest: 3000, host: 3000, auto_correct: true
+  config.vm.network "forwarded_port", guest: 80, host: 8080, auto_correct: true
+  
+  config.vm.provider "virtualbox" do |vb|
+   vb.memory = "1024"
+  end
+
+  ## SETTING UP BASE SYSTEM ##
+  config.vm.provision "shell", privileged: true, inline: <<-SHELL
+    apt-get -y update
+    apt-get -y install build-essential
+    apt-get -y install zlib1g-dev git-core nodejs
+
+    # Install MySQL
+    debconf-set-selections <<< "mysql-server-5.5 mysql-server/root_password password root"
```

Comment:
Can we change this to `mysql-server mysql-server/root_password` so that it is not dependent on the mysql version? I kind of surprises me the config works even though mysql 5.7 is installed.